frontend/src/metabase/common/components/FieldPicker/FieldPicker.tsx

```diff
@@ -34,25 +43,23 @@ export const FieldPicker = ({
   onSelectAll,
   onSelectNone,
   isColumnSelected,
+  isColumnDisabled = () => false,
```

Comment:
`isColumnDisabled` has a different reference on every render when the default value is applied which breaks memoization of `items`

```suggestion
  isColumnDisabled = DEFAULT_IS_COLUMN_DISABLED,
```

Reply:
Fixed